guides/source/developers/products-and-variants/product-images.html.md

```diff
@@ -41,25 +41,29 @@ for a specific variant or for **All** variants.
 If set to **All**, the `viewable_id` is set to the master variant for the
 current product.
 
-## Paperclip settings
+## Image settings
 
-[Paperclip][paperclip-gem] handles the creation and storage of product images.
-By default, it creates several versions of each image at specific sizes.
+[Active Storage][activestorage] and [Paperclip][paperclip-gem] handle the
+creation and storage of product images.  By default, they creates several
```

Comment:
Not the endo of the world, but there's an extra space before `By`

Reply:
I replaced all the occurrences with a single space 👍 

This is something I do by default when writing.